EIPS/eip-2981.md

```diff
@@ -131,15 +135,19 @@ This EIP does not specify the manner of payment to the royalty recipient. Furthe
 
 This EIP does not specify how the `_royaltyAmount` is calculated from the sale price `_value`. Many NFT contracts may follow a fixed-percentage royalty that is the same regardless of price. However, EIP-2981 implementers may choose whatever logic they want to calculate the `_royaltyAmount` - this EIP does not mandate a fixed-percentage fee model.
 
-Additionally, we return a specific `_royaltyAmount` so there is no dispute with a marketplace over how much is owed for a given sale price.
+Additionally, we return a specific `_royaltyAmount`, which must be paid in the same token the sale (`_value`), so there is no dispute with a marketplace over how much is owed for a given sale price.
 
 ### Unused data parameters in function signature
 
 This EIP mandates the inclusion of the `_data` parameter and the `_royaltyPaymentData` return value, and also mandates that these are unused. The purpose of this is to have a seamless path to build on this ERC by further EIP standards that specify payment notifications, fee splitting, and other advanced use cases.
 
 ## Backwards Compatibility
 
-This standard is completely compatible with current ERC-721 standards - in fact it requires it.
+This standard must implement ERC-165 to allow introspection, but it does not rely on, or extend, any other existing standard. However, there are already proprietary methods for royalty management used by existing marketplaces, so it will require additional re-work on the part of marketplaces.
```

Comment:
NFTs that support this standard must implement ERC-165 to allow introspection, ...